vali_objects/vali_dataclasses/order.py

```diff
@@ -9,12 +9,15 @@
 from vali_objects.vali_dataclasses.order_signal import Signal
 from enum import Enum, auto
 
+ORDER_SRC_ORGANIC = 0
+ORDER_SRC_ELIMINATION_FLAT = 1
```

Comment:
nit: could be an OrderSrc enum

overall looks good, adds order on elimination and handles position rebuild appropriately